scholarly_articles/scripts/set_country.py

```diff
@@ -5,9 +5,19 @@
 
 
 def run():
+    #first iteration to identify country by name
     for country in Country.objects.all():
         for aff in models.Affiliations.objects.filter(
                 Q(name__icontains=country.name_en) | Q(name__icontains=country.name_pt),
                   country__isnull=True, official__isnull=True).iterator():
             aff.country = country
             aff.save()
+
+    #second iteration to identify country by acronym with 3 char
+    for country in Country.objects.all():
```

Comment:
@Rossi-Luciano pode remover a linha do segundo `for`, assim reaproveita o primeiro `for`